src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2702,7 +2711,11 @@ MavlinkReceiver::receive_start(pthread_t *thread, Mavlink *parent)
 	param.sched_priority = SCHED_PRIORITY_MAX - 80;
 	(void)pthread_attr_setschedparam(&receiveloop_attr, &param);
 
+#ifdef CONFIG_NET
```

Comment:
This would be better structured as 
```
#if !defined(CONFIG_NET)
#  define NET_ADDED_STACK 0
#  else 
#  define NET_ADDED_STACK 1360
#endif
```

At the top of file or header
and  use
`pthread_attr_setstacksize(&receiveloop_attr, PX4_STACK_ADJUSTED(2840+NET_ADDED_STACK ));`
here